kalite/caching/__init__.py

```diff
@@ -65,7 +68,11 @@ def invalidate_all_caches():
     call in here.
     """
     invalidate_inmemory_caches()
-    initialize_content_caches(force=True)
+    if settings.DO_NOT_RELOAD_CONTENT_CACHE_AT_STARTUP:
```

Comment:
I don't really understand _why_ this is necessary, can you help me make the connection in plain English? If we `DO_NOT_RELOAD_CONTENT_CACHE_AT_STARTUP`, then we should remove all these `*.cache` files, but why?

On the other hand, if that setting is false then we should `initialize_content_caches`, which creates in-memory (as opposed to disk-based) caches?